### PR TITLE
Update presentations on website

### LINKED ISSUE #11: Format publications and add posters

### PR DESCRIPTION
**Pull request by siwelwerd**
Closes #11

### CODE REVIEW

File: site/pages/about.md [hunk 1]
```diff
@@ -18,17 +18,17 @@ If you're planning to use TBIL in your classroom, you can get a link to join our
...
 
     - Team-Based Learning (TBL) is a cooperative learning strategy blending elements of flipped learning, inquiry-based learning, and problem-based learning. Although used quite frequently in other disciplines, use of this strategy in mathematics has been limited. In this article, we describe how TBL can be implemented in math courses with adherence to essential elements of TBL and introduce modifications specific to mathematics instruction. In particular, we introduce a particular style of TBL, which we term Team-Based Inquiry Learning, that satisfies the defining pillars of inquiry-based learning.
 
 
-- Lewis, D. and Estis, J. 2020. Improving Mathematics Content Mastery and Enhancing Flexible Problem Solving through Team-Based Inquiry Learning. Teaching & Learning Inquiry 8 (2), 165-183. <https://doi.org/10.20343/teachlearninqu.8.2.11>
+- Lewis, D. and Estis, J. (2020). Improving Mathematics Content Mastery and Enhancing Flexible Problem Solving through Team-Based Inquiry Learning. _Teaching & Learning Inquiry_ 8 (2), 165-183. <https://doi.org/10.20343/teachlearninqu.8.2.11>
```
Comment (jkostiuk) [inline site/pages/about.md:26]: This link isn't working for me today---possibly a ucalgary issue?

File: site/pages/about.md [hunk 2]
```diff
@@ -40,14 +40,20 @@ If you're planning to use TBIL in your classroom, you can get a link to join our
 
 - Kate Owens† has created [these excellent slides](https://docs.google.com/presentation/d/1824oMy2HetUVz4AlEZFufjftg72gh0Zqqu4RzOP08PE/edit?usp=sharing) overviewing TBIL and resources to implement it in your classroom.
 
+- Drew Lewis presented a poster "[Transforming Lower Division Mathematics Through Team-Based Inquiry Learning](https://drive.google.com/file/d/1rruYnUEJKJZROXcXGkjl7aXdtc2v5FSs/view?usp=drive_link)" at the IUSE Summit (June 2022)
```
Comment (jkostiuk) [inline site/pages/about.md:43]: poster isn't publicly available in Drive

**Review by jkostiuk — CHANGES_REQUESTED** [review]
two links to fix (though not clear how to fix the ucalgary one?)

File: site/pages/about.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (siwelwerd) [reply site/pages/about.md:43]: @jkostiuk Thanks for catching that, try this link now and commit the suggestion if it works

File: site/pages/about.md (hunk 1 above)
Comment (siwelwerd) [reply site/pages/about.md:26]: Yeah, I think just leave this one alone and it's really on the journal to fix their website.

**Review by jkostiuk — APPROVED** [review]

**Pull request merged**